### PR TITLE
fix address parsing issue

### PR DESCRIPTION
**Pull request by codemaestro64**
This solves the issue of displaying addresses as testnet addresses regardless of the current net. It solves it by fetching current net correctly which was the issue before now.

### CODE REVIEW

File: bchain/coins/dcr/decredparser.go [hunk 1]
```diff
@@ -54,7 +61,12 @@ func GetChainParams(chain string) *chaincfg.Params {
 			panic(err)
```
Comment (dmigwi) [inline bchain/coins/dcr/decredparser.go]: I feel like this part of the code is beats the purpose when using the testnet3.
```
if !chaincfg.IsRegistered(&MainNetParams) {
	err := chaincfg.Register(&MainNetParams)
	if err != nil {
		panic(err)	
	}
}
```

File: bchain/coins/dcr/decredparser.go [hunk 2]
```diff
@@ -54,7 +61,12 @@ func GetChainParams(chain string) *chaincfg.Params {
...
 		}
 	}
+
 	switch chain {
+	case "testnet3":
+		return &TestNetParams
+	case "mainnet":
```
Comment (dmigwi) [inline bchain/coins/dcr/decredparser.go]: The case isn't necessary.

Comment (dmigwi) [reply bchain/coins/dcr/decredparser.go]: I mean the `case "mainnet":` isn't necessary as there are only two case to consider.

**Comment by codemaestro64** [comment]
Closing this in favor of #2

**Pull request closed without merging**